src/solvers/CMakeLists.txt

```diff
@@ -98,6 +98,36 @@ elseif("${sat_impl}" STREQUAL "glucose")
     target_sources(solvers PRIVATE ${glucose_source})
 
     target_link_libraries(solvers glucose-condensed)
+elseif("${sat_impl}" STREQUAL "cadical")
+    message(STATUS "Building solvers with cadical")
+
+    download_project(PROJ cadical
+        URL https://github.com/arminbiere/cadical/archive/sr19.tar.gz
+        PATCH_COMMAND true
+        COMMAND CXX=${CXX_COMPILER} CXXFLAGS=-O3 ./configure --debug
```

Comment:
Double-check: is the `--debug` asking `configure` for verbose output, or setting `-g` for the build, or could it even be overriding the `-O3` given in `CXXFLAGS` to get `-g -O0`?

Reply:
I did double-check when building locally: it just adds `-g`, the `-O3` from `CXXFLAGS` is passed on as desired.